### DISCUSSION TITLE
MultiLanguage Support

### DISCUSSION THREAD

**Discussion by RiemaruKarurosu** [noisetorch/NoiseTorch#306] [Ideas (I would suggest ...)]
Suggestions
Create Different Readme version to handle different language support, and let the community upload their different lang support.

**Comment by Foexle11** [comment]
I've seen .ini files that somehow take the Input there to change certain words. That was for me an easy way as someone with not much coding expirience to create my own language-file. Would that be an option here?

**Comment by Technetium1** [comment]
I've had positive experiences with [Crowdin](https://crowdin.com/pricing#annual) in the past: https://support.crowdin.com/github-integration/
